bedrock/firefox/templates/firefox/browsers/mobile/get-ios.html

```diff
@@ -0,0 +1,28 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends 'landing/base-landing.html' %}
+
+{% block page_title %}{{ ftl('get-ios-firefox-for-ios') }}{% endblock %}
+{% block page_desc %}{{ ftl('get-ios-firefox-is-fast') }}{% endblock %}
+{% block page_favicon %}{{ static('img/favicons/firefox/browser/favicon.ico') }}{% endblock %}
+{% block page_favicon_large %}{{ static('img/favicons/firefox/browser/favicon-196x196.png') }}{% endblock %}
+{% block page_ios_icon %}{{ static('img/favicons/firefox/browser/apple-touch-icon.png') }}{% endblock %}
+
+{% block icon_class %}mzp-t-product-firefox{% endblock %}
+{% block icon_text %}ksdfjh{% endblock %}
+{% block header %}{{ ftl('get-ios-download-firefox') }}{% endblock %}
+{% block body_copy %}{{ ftl('get-ios-firefox-mobile-adapts') }}{% endblock %}
+{% block ios_badge %}
+  <a class="ios store-badge" id="appStoreLink-primary" href="https://app.adjust.com/jcn6eyw?campaign=firefox100&adgroup=ios-store" data-link-type="download" data-download-os="iOS">
```

Comment:
We will need a unique adjust link to be placed here, this will be removed when we get it from the data team